src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,105 @@
+@use "../../variables" as *;
+.footer {
+  text-align: center;
+  margin: 0px;
+  padding-top: 36px;
+  .information {
+    text-align: center;
+    padding-bottom: 52px;
```

text-alignが不要に見えるので修正お願いします
```suggestion
    padding: 52px 0;
```